sdk/translation/Azure.AI.Translation.Document/tests/Infrastructure/DocumentTranslationLiveTestBase.cs

```diff
@@ -65,66 +65,87 @@ public BlobContainerClient GetBlobContainerClient(string containerName)
-            }
+        public Uri CreateTargetContainer(List<TestDocument> documents = default)
+        {
+            var containerClient = CreateContainer("target", documents);
+            var expiresOn = DateTimeOffset.UtcNow.AddHours(1);
+            return containerClient.GenerateSasUri(BlobContainerSasPermissions.List | BlobContainerSasPermissions.Write, expiresOn);
+        }
 
+        public async Task<Uri> CreateTargetContainerAsync(List<TestDocument> documents = default)
+        {
+            var containerClient = await CreateContainerAsync("target", documents);
             var expiresOn = DateTimeOffset.UtcNow.AddHours(1);
             return containerClient.GenerateSasUri(BlobContainerSasPermissions.List | BlobContainerSasPermissions.Write, expiresOn);
         }
+        public async Task<Uri> CreateGlossaryAsync(TestDocument document = default)
+        {
+            var containerClient = await CreateContainerAsync("glossary", new List<TestDocument> { document });
+            var expiresOn = DateTimeOffset.UtcNow.AddHours(1);
+            var glossaryContainerSasUri = containerClient.GenerateSasUri(BlobContainerSasPermissions.Read | BlobContainerSasPermissions.List, expiresOn);
+            return new Uri(String.Format("{0}{1}{2}{3}/{4}{5}", glossaryContainerSasUri.Scheme, Uri.SchemeDelimiter, glossaryContainerSasUri.Authority, glossaryContainerSasUri.AbsolutePath, document.Name, glossaryContainerSasUri.Query));
+        }
+
+        public Uri CreateGlossary(TestDocument document = default)
+        {
+            var containerClient = CreateContainer("glossary", new List<TestDocument> { document });
+            var expiresOn = DateTimeOffset.UtcNow.AddHours(1);
+            var glossaryContainerSasUri = containerClient.GenerateSasUri(BlobContainerSasPermissions.Read | BlobContainerSasPermissions.List, expiresOn);
+            return new Uri(String.Format("{0}{1}{2}{3}/{4}{5}", glossaryContainerSasUri.Scheme, Uri.SchemeDelimiter, glossaryContainerSasUri.Authority, glossaryContainerSasUri.AbsolutePath, document.Name, glossaryContainerSasUri.Query));
+        }
+
         public async Task<Tuple<Uri, BlobContainerClient>> CreateTargetContainerWithClientAsync(List<TestDocument> documents = default)
+        {
+            var containerClient = await CreateContainerAsync("target", documents);
+            var expiresOn = DateTimeOffset.UtcNow.AddHours(1);
+            var containerSasUri = containerClient.GenerateSasUri(BlobContainerSasPermissions.List | BlobContainerSasPermissions.Write, expiresOn);
+            return Tuple.Create(containerSasUri, containerClient);
+        }
+
+        private BlobContainerClient CreateContainer(string name, List<TestDocument> documents)
         {
             Recording.DisableIdReuse();
-            var containerName = "target" + Recording.GenerateId();
+            var containerName = "source" + Recording.GenerateId();
```

Comment:
looks like this should be `var containerName = name + Recording.GenerateId();`

Reply:
or you could just use your `GenerateRandomName` function

Reply:
oh that would've been a disaster. Missed that when I was merging. Thanks